features/step_definitions/generic_front_end_api.rb

```diff
@@ -0,0 +1,3 @@
+Then(/^the text "([^"]*)" is displayed on the page$/) do |search_text|
+  page.should have_content(search_text)
+end
```

Comment:
Think the name of the file should be just generic_front_end.rb (i.e. without the api bit as it's just relating to the frontend)